document_merge_service/settings.py

```diff
@@ -28,6 +28,24 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=default(["*"]))
 
 
+# Database
+# https://docs.djangoproject.com/en/1.11/ref/settings/#databases
+
+DATABASES = {
+    "default": {
+        "ENGINE": env.str("DATABASE_ENGINE", default="django.db.backends.sqlite3"),
+        "NAME": env.str(
+            "DATABASE_NAME", default="/var/lib/document-merge-service/data/sqlite3.db"
+        ),
+        "USER": env.str("DATABASE_USER", default=""),
+        "PASSWORD": env.str("DATABASE_PASSWORD", default=""),
+        "HOST": env.str("DATABASE_HOST", default=""),
+        "PORT": env.str("DATABASE_PORT", default=""),
+        "OPTIONS": env.dict("DATABASE_OPTIONS", default={}),
+    }
+}
+
+
```

Comment:
Why the move of the `DATABASES` block?

Reply:
I need to be able to access DATABASES['default']['ENGINE'] to be able to determine whether postgres needs to be added to `INSTALLED_APPS`.